at_onboarding_flutter/lib/screen/at_onboarding_activate_screen.dart

```diff
@@ -23,12 +23,16 @@ class AtOnboardingActivateScreen extends StatefulWidget {
   final bool hideReferences;
   final String? atSign;
   final AtOnboardingConfig config;
+  final OnboardingService onboardingService;
+  final FreeAtsignService freeAtsignService;
 
   const AtOnboardingActivateScreen({
     Key? key,
     required this.hideReferences,
     this.atSign,
     required this.config,
+    required this.onboardingService,
```

Comment:
New required params, will this impact any existing functionality?

Reply:
No. I've tested the full functionality, these are working well.